src/mixins/bcol-mixin.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Vue } from 'vue-property-decorator'
+import axios from '@/axios-auth'
+
+@Component({})
+export default class BcolMixin extends Vue {
+  async getErrorObj (errCode) {
+    const fetchUrl = sessionStorage.getItem('PAY_API_URL') + 'codes/errors/' + errCode
+    const errObj = await axios.get(fetchUrl)
```

Comment:
You need to handle network error exceptions. You could use `.catch()`, or wrap this in a try..catch block, or handle this in the calling code.

For an example of this, see https://github.com/bcgov/bcrs-business-create-ui/blob/7a405e7c1a14fa821a8cf4df8dca34ffd55248ba/src/components/common/Actions.vue#L118

Reply:
done

Reply:
I thought you might handle the error instead of ignoring it. If the intent is truly to ignore the error (and I don't have a problem with this) then please add a comment that you're explicitly ignoring the error, so it doesn't look like an oversight. Thanks.

Reply:
Fixed.